apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -77,6 +83,10 @@ $(document).ready(() => {
     optimizelyCountUserTypeSelection(getUserType());
     // Optimizely-related code for new sign-up user-type buttons (end)
 
+    // Optimizely-related code for teacher opting to share email with regional partner (start)
+    optimizelyCountSuccessSignupWithRegPartnerOpt();
```

Comment:
Out of attempt to fully understand, is this only being called when they signup and opt in or for every signup?

Reply:
This one is for every signup just to compare rates of signups between those that see the opt-in option and thsoe that don't in the A/B experiment. We just want to make sure the addition of this new choice doesn't statistically decrease the successful rates of signups.

Reply:
Ahh makes sense. Thanks Turner!